packages/core/src/themes/global.ts

```diff
@@ -1,7 +1,7 @@
 import { FontFace, GlobalSheet } from 'aesthetic';
 import { Theme } from '../types';
 
-export default (fontFaces: { [fontFamily: string]: FontFace[] }) => ({ color }: Theme) =>
+export default (fontFaces: { [fontFamily: string]: FontFace[] }) => ({ color, font }: Theme) =>
   ({
     '@global': {
       '*': {
```

Comment:
would this be a good time to finally add:

```css
:focus:not(:focus-visible) { outline: none }
```

Gets rid of the annoying outline for mouse users but preserves it for keyboard users, and is ignored by browsers that don’t support `:focus-visible`.

Reply:
i see you've added it! resolving this!